api/odilo.py

```diff
@@ -0,0 +1,532 @@
+# coding=utf-8
+import datetime
+import json
+from flask.ext.babel import lazy_gettext as _
+
+from circulation import (
+    LoanInfo,
+    HoldInfo,
+    FulfillmentInfo,
+    BaseCirculationAPI,
+)
+from core.odilo import (
+    OdiloAPI as BaseOdiloAPI,
+    OdiloRepresentationExtractor,
+    OdiloBibliographicCoverageProvider,
+    MockOdiloAPI as BaseMockOdiloAPI,
+)
+
+from core.model import (
+    Credential,
+    DataSource,
+    DeliveryMechanism,
+    ExternalIntegration,
+    Identifier,
+    Representation,
+)
+
+from core.monitor import (
+    CollectionMonitor,
+)
+from core.util.http import HTTP
+
+from circulation_exceptions import *
+
+
+class OdiloAPI(BaseOdiloAPI, BaseCirculationAPI):
+    NAME = ExternalIntegration.ODILO
+    DESCRIPTION = _("Integrate an Odilo library collection.")
+    SETTINGS = [
+                   {"key": BaseOdiloAPI.LIBRARY_API_BASE_URL, "label": _("Library API base URL")},
```

Comment:
This is where you would set DEFAULT_API_BASE_URL as a `default`.

Reply:
yes, I realized